core/logger/logger_test.go

```diff
@@ -176,13 +201,53 @@ func TestWithFileRotation(t *testing.T) {
 	file := log.File()
 	assert.NotNil(t, file)
 }
+
 func TestWithStderrRedirect(t *testing.T) {
 	log := logger.DefaultLogger()
 
 	err := log.ApplyOptions(logger.WithStderrRedirect())
 	assert.Nil(t, err)
 }
 
+func TestHeadline(t *testing.T) {
+	log := logger.DefaultLogger()
+	message := "One blockchain to rule them all"
+	delimiter := "///"
+	timestamp := "-"
+
+	formatedMessage := log.Headline(message, delimiter, timestamp)
+	assert.Contains(t, formatedMessage, message)
+	assert.Contains(t, formatedMessage, delimiter)
+	assert.Contains(t, formatedMessage, timestamp)
+}
+
+func TestRedirectStderr(t *testing.T) {
```

Comment:
I think we should also test that it actually writes a panic where we redirect it

Reply:
Done.